### ISSUE TITLE
Global widgets, template and sections - seperate content from style

### ISSUE THREAD

**Issue by danstramer** [pojome/elementor#3374]
The Global widget, template and sections are great, but they are missing a very important feature - to be able to reuse them and just change the content but keep the styling global.
For instance, I create a title widget and make it global. This title would have font size, color, padding, border applied to it and of course the title itslef.
I would then drag this widget to a page I would like to use it on, but in this page I would like to change the title, just the title, not the style. I can't do that without it changing all the other titles (if I edit the global widget), the second option would be to unlink the widget but then I would lose the styling capabilities if I would like to change the font size, color etc...

So the solution would be enabling content edits without losing the global styling.
Another feature for this would be locking the styling for a user role (example: editor), and not allowing them to make any style changes, just content changes.
I think this is a very important feature which resides with the notion of separating content from structure.

Would love to hear your thoughts about this,

Thanks
Dan
Then I would

**Comment by bainternet** [comment]
@danstramer 

We are working on something that will let you do just that.

as for the user role locking, ref: #1314

**Comment by danstramer** [comment]
Thanks Ohad. 
Any eta for this and the role manager?

Dan

**Comment by bainternet** [comment]
Sadly no eta that I know of but it is one of the next features to make it in.

**Comment by danstramer** [comment]
Thanks!

**Comment by danstramer** [comment]
Hi, any updates on this?

Thanks
Dan

**Comment by danstramer** [comment]
Thanks @bainternet , but I was referring to the issue of creating global widgets and being able to edit only the content in them without detaching them: Having global widgets with different content but being able to make a design change and having the design propagate through all of the instances of that global widget.

**Comment by danstramer** [comment]
Thank, I think that separating design from content would be super helpful.
After creating 20 CTA widgets (same design but different content) if I would like a design change today I would need to go through each one of them.
Having a global widget that would hold the design but will enable to have different content would be a huge time saver and a logical way to go.

Dan

**Comment by danstramer** [comment]
Hi, any update regarding: 
"creating global widgets and being able to edit only the content in them without detaching them: Having global widgets with different content but being able to make a design change and having the design propagate through all of the instances of that global widget."

Thanks!
Dan

**Comment by thomasmalley** [comment]
+1 Why is this closed?

**Comment by thomasmalley** [comment]
@bainternet This was not solved by the Role Manager; Because it is a completely different application and could Speed up the Design Process a lot! 

I see different use cases for Client work: 
- The Client/Customer wants a last minute Change to one particular color. I used global widgets to speed up the design Process but had to unlink most of them to change the Content according to the Context. Now I lost the global styling capability.. Sure there is still css and if I gave a class to the Global Widget beforehand I could use that to make a last minute Change even though the widgets aren't global any more but that should be an option included into elementor in my Opinion. 

Like @danstramer said:

> I think this is a very important feature which resides with the notion of separating content from structure.
